packer-templates/http/user-data

```diff
@@ -1,10 +1,11 @@
-#cloud-config
+# cloud-init-modules-config
+# https://cloudinit.readthedocs.io/en/latest/reference/modules.html
 autoinstall:
   version: 1
   locale: en_US
   keyboard:
-    layout: en
-    variant: us
+    layout: us
+    variant: ""
```

Comment:
```suggestion
```

Not required according to https://cloudinit.readthedocs.io/en/latest/reference/modules.html#keyboard